package.json

```diff
@@ -92,15 +92,14 @@
     "terra-markdown": "^2.0.0",
     "terra-slide-panel": "^2.0.1",
     "terra-theme-provider": "^2.0.0",
-    "terra-toolkit": "^3.0.0",
     "to-ast": "^1.0.0",
-    "webpack-merge": "^4.1.2",
```

Comment:
This is a dependency included in toolkit

Reply:
We should still include it as a dependency here because we use it directly.

Reply:
Does this mean we should not have included webpack-merge as a dependency of toolkit? it was included as a dependency in [v3 as well](https://github.com/cerner/terra-toolkit/pull/113/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R97)

But I will go ahead and make this change.